app/Http/Controllers/TaskController.php

```diff
@@ -90,6 +90,8 @@ public function update(Request $request, $id)
      */
     public function destroy($id)
     {
-        //
+        $task = Task::find($id);
```

Comment:
dùng findorfail được k? khác nhau như nào? 
xử lý nếu không tìm thấy

Reply:
Dạ có thể dùng hàm findOrFail() ạ vì hàm findOrFail() sẽ tìm task trong database với id truyền vào, nếu có thì sẽ trả về task đó còn nếu không tìm thấy thì sẽ báo lỗi không tìm thấy 404

Reply:
xử lý trường hợp không tìm thấy

Reply:
dạ em sẽ báo lỗi ạ

Reply:
em dùng try catch để bắt lỗi rồi sau đó e custom lại lỗi 404 đó với cái message là task does not exists rồi sau đó gọi ra ạ